lib/Generator.js

```diff
@@ -36,6 +36,7 @@
  * @property {NormalModule} module the module
  * @property {ChunkGraph} chunkGraph
  * @property {RuntimeSpec} runtime
+ * @property {RuntimeTemplate=} runtimeTemplate
```

Comment:
maybe we need to add `UpdateHashContext` for module and chunk separately. right now this is copy-paste of `UpdateHashContext` in `lib/Dependency`